packages/web-app-files/tests/unit/components/AppBar/ViewOptions.spec.ts

```diff
@@ -52,8 +53,8 @@ describe('ViewOptions component', () => {
 
 function getWrapper(
   { perPage = '100' } = {},
-  props: { [key: string]: any } = {
-    viewModes: []
+  props?: {
+    viewModes: ViewMode[]
```

Comment:
Unused

Reply:
Not sure how to resolve this - my IDE complains in line 46 of the same file if I remove the type annotations. If you don't mind and there's no other issues let's get this merged and let me continue, not worth another 45min of acceptance test pipelines

Reply:
oh misread the diff, nevermind.